.github/workflows/ci.yml

```diff
@@ -5,7 +5,6 @@ on:
     paths:
       - '.github/workflows/ci.yml'
       - '.swiftlint.yml'
-      - '**/*.swift'
```

Comment:
This is the paths that the build triggers on, set it to all files by just deleting the "paths" part. Deleting `**/*.swift` will make the build not trigger on anything except the `.swiftlint.yml` and the `ci.yml`